src/command/stage.ts

```diff
@@ -43,7 +43,7 @@ export async function unstage(
         args.push(...['rm', '--cached', '-r', '--']);
     } else {
         if (_where === 'working-tree') {
-            args.push(...['checkout-index', '-f']);
+            args.push(...['checkout-index', '-f', '-u']);
```

Comment:
Otherwise, it fails with a `git checkout-index: HEAD is not in the cache` error.

Reply:
Also: https://stackoverflow.com/questions/43239029/what-does-git-checkout-index-u-or-index-do